### PR TITLE
Padietl huntsville

### PR DESCRIPTION
**Pull request by pdietl**
- :middle_finger: 
    -  :micronesia:

### CODE REVIEW

**Comment by KyleBostian** [comment]
Leave a general comment

**Comment by jmhossler** [comment]
sup

File: padietl-huntsville.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Huntsville
+
+### Places to Eat
+- 88 Buffet
+- Rosie's
+- ADTRAN Cafeteria
+
```
Comment (jmhossler) [inline padietl-huntsville.md]: You need a lot more places to eat

File: padietl-huntsville.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Huntsville
+
+### Places to Eat
```
Comment (jmhossler) [inline padietl-huntsville.md:3]: Add more restaurants

File: padietl-huntsville.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- 88 Buffet
+- Rosie's
+- ADTRAN Cafeteria
```
Comment (KyleBostian) [inline padietl-huntsville.md:6]: Best place in town. Great recommendation.

File: padietl-huntsville.md (hunk 1 above)
Comment (jmhossler) [reply padietl-huntsville.md]: Taco Bell, I Love Korea, Five Guys, Mangos, etc

Comment (pdietl) [reply padietl-huntsville.md]: IK

**Review by jmhossler — APPROVED** [review]

**Review by KyleBostian — APPROVED** [review]
Well done. 🚢

**Pull request merged**